frontend/src/metabase/admin/settings/components/SettingsEmailForm.jsx

```diff
@@ -184,10 +183,14 @@ export default class SettingsEmailForm extends Component {
             let errorMessage = (formErrors && formErrors.elements) ? formErrors.elements[element.key] : validationErrors[element.key];
             let value = formData[element.key] == null ? element.defaultValue : formData[element.key];
 
-            return <SettingsEmailFormElement
-                        key={element.key}
-                        element={{ ...element, value, errorMessage }}
-                        handleChangeEvent={this.handleChangeEvent.bind(this)} />
+            return (
+                <SettingsSetting
```

Comment:
SettingsSetting isn't my favorite component name in the world but to be fair I can't think of a better name.